includes/ajax.php

```diff
@@ -61,7 +61,7 @@ function tsml_ajax_groups() {
 				'contact_3_phone'	=> @$group_custom['contact_3_phone'][0],
 				'last_contact'		=> @$group_custom['last_contact'][0],
 				'notes'				=> $group->post_content,
-				'tokens'			=> tsml_string_tokens($title),
+				'tokens'			=> tsml_string_tokens($group->post_title),
```

Comment:
this looks like it's for a separate issue right?

Reply:
Yes, I had some changes in this branch that I'd forgotten about.  Would it be better if I roll back anything not having to do with the ACF issue and push another commit?

Reply:
Nah, if it needs to be fixed I say let's fix it :)

Reply:
Cool. his released to any of our other open issues?